kernel/init.c

```diff
@@ -142,6 +142,10 @@ void _bss_zero(void)
 {
 	memset(&__bss_start, 0,
 		 ((u32_t) &__bss_end - (u32_t) &__bss_start));
+#ifdef CONFIG_CCM_BASE_ADDRESS
```

Comment:
I think this should be moved into _PrepC.

Reply:
@galak  I put it there because the BSS init is also in that location. Both BSS and CCM init have almost the same function, so shouldn't they be close together ?